src/web/routes.rs

```diff
@@ -29,7 +29,8 @@ pub(super) fn build_routes() -> Routes {
         PermanentRedirect("/-/static/opensearch.xml"),
     );
 
-    routes.static_resource("/-/static/:file", super::statics::static_handler);
+    routes.static_resource("/-/static/:single", super::statics::static_handler);
+    routes.static_resource("/-/static/*", super::statics::static_handler);
```

Comment:
Can you use `/-/static/*` for both of these? I would expect it to work for single files too.

Reply:
I'm not sure why but just `/-/static/*` did not work when I tried it :/

I can check again...

Reply:
It's probably hitting the bugs around priority of the different specificity levels in the old version of `route-recogniser`

Reply:
Chatted with @Nemo157 on this a bit on Discord. It's a bug of the router library, and this workaround seems to work.